.gitignore

```diff
@@ -1,6 +1,5 @@
 _b0
 _build
-src/uunf_data.ml
```

Comment:
I wonder, is there maybe a way to generate this as part of a `dune` rule instead of running `pkg.ml`?

Reply:
That would certainly be possible. I chose to check in the result of running `pkg.ml` instead as that program downloads character data and then uses the data to generate some code and I didn't want the build target to depend on an internet connection if we could avoid it.